config/locales/ru.yml

```
@@ -116,4 +116,34 @@ ru:
   enable_hints: 'Включить подсказки?'
   max_hints_count: 'Количество доступных подсказок'
   loading_please_wait: 'Загрузка... Подождите...'
-  choose_another_answer: 'Неверно, выберите другой ответ.'
+  choose_another_answer: 'Неверно, выберите другой ответ.'  
+  edit: 'Редактировать'
+  delete: 'Удалить' 
+  actions: 'Действия'
+  add: 'Добавить'
+  remove: 'Убрать'
+  available_tasks: 'Доступные задачи'
+  reset: 'Сбросить'
+  email: 'Почта'
+  
+  created_at: 'Дата создания'
+  
+  
+  users:
+    role: 'Роль'
+    edit_users: 'Редактировать пользователей'
+    users_list: 'Список пользователей'
+    confirmation_deletion_user: 'Вы уверены, что хотите удалить пользователя?'
+
+  tags:
+    new_tag: 'Новый тэг'
+    my_tags: 'Мои тэги'
+    new_task_tag: 'Новый тег для задачи'
+    tags_name: 'Имя тега'
+    create_tag: 'Создание тега'
+    enter_tag_name: 'Введите название'
+    filter_by_tags: 'Фильтр по тегам'
+    confirmation_deletion_tag: 'Вы уверены, что хотите удалить тег?'
+    tag_exist: 'Тег с таким названием уже существует'
+
+  
```

в самом конце файла теперь строка с пробелами, такого быть не должно
```suggestion
    tag_exist: 'Тег с таким названием уже существует'

```

---

Исправлено